docs/sonata-simulation.rst

```diff
@@ -120,17 +120,21 @@ Here's a quick list of them which can be used as inspiration:
      - Circuit config
      - Node Sets
    * - Hippocampus
-     - `simulation_config.json <https://bbpgitlab.epfl.ch/hpc/sim/blueconfigs/-/blob/main/sonataconf-quick-hip-multipopulation/simulation_config.json>`__
-     - `circuit_config.json <https://bbpgitlab.epfl.ch/hpc/sim/blueconfigs/-/blob/main/sonataconf-quick-hip-multipopulation/circuit_config.json>`__
-     - `node_sets.json <https://bbpgitlab.epfl.ch/hpc/sim/blueconfigs/-/blob/main/sonataconf-quick-hip-multipopulation/node_sets.json>`__
+     - `simulation_config.json <https://github.com/BlueBrain/blueconfigs/blob/main/sonataconf-quick-hip-multipopulation/simulation_config.json>`__
+     - `circuit_config.json <https://github.com/BlueBrain/blueconfigs/blob/main/sonataconf-quick-hip-multipopulation/circuit_config.json>`__
+     - `node_sets.json <https://github.com/BlueBrain/blueconfigs/blob/main/sonataconf-quick-hip-multipopulation/node_sets.json>`__
    * - SSCX
-     - `simulation_config.json <https://bbpgitlab.epfl.ch/hpc/sim/blueconfigs/-/tree/main/sonataconf-quick-scx-multi-circuit/simulation_config.json>`__
-     - `circuit_config.json <https://bbpgitlab.epfl.ch/hpc/sim/blueconfigs/-/tree/main/sonataconf-quick-scx-multi-circuit/circuit_config.json>`__
-     - `node_sets.json <https://bbpgitlab.epfl.ch/hpc/sim/blueconfigs/-/tree/main/sonataconf-quick-scx-multi-circuit/node_sets.json>`__
+     - `simulation_config.json <https://github.com/BlueBrain/blueconfigs/tree/main/sonataconf-quick-scx-multi-circuit/simulation_config.json>`__
+..  TODO: circuit_config.json link does not exist for now. Fix it
+
+     - `circuit_config.json <https://github.com/BlueBrain/blueconfigs/tree/main/sonataconf-quick-scx-multi-circuit/circuit_config.json>`__
+     - `node_sets.json <https://github.com/BlueBrain/blueconfigs/tree/main/sonataconf-quick-scx-multi-circuit/node_sets.json>`__
    * - Thalamus
-     - `simulation_config.json <https://bbpgitlab.epfl.ch/hpc/sim/blueconfigs/-/blob/main/sonataconf-quick-thalamus/simulation_config.json>`__
+     - `simulation_config.json <https://github.com/BlueBrain/blueconfigs/tree/main/sonataconf-quick-thalamus/simulation_config.json>`__
```

Comment:
I propose to remove that thalamus example. Its `circuit_config.json` used by the `blueconfigs` repo is the file stored in `gpfs proj12` space, which is not available anymore.